configure.ac

```diff
@@ -1100,6 +1100,31 @@ else
     AC_DEFINE([HAVE_SYS_RANDOM_H],0,[Define to 1 if you have the <sys/random.h> header file.])
 fi
 
+HAVE_GETENTROPY=
+AC_LANG_PUSH([C++])
```

Comment:
Why is this needed?

Reply:
I realize that there is no need to check the script for C++ compiler. Man page of autoconf (1) says `configure scripts check for the C compiler and its features by default.`. Then, imho, `AC_LANG_PUSH([C++])` is not necessary so I removed it.

(1): [https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Language-Choice.html](https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Language-Choice.html)